experiment/stop_experiment.py

```diff
@@ -29,8 +30,15 @@ def stop_experiment(experiment_name, experiment_config_filename):
     experiment_config = yaml_utils.read(experiment_config_filename)
 
     if experiment_config.get('local_experiment', False):
-        raise NotImplementedError(
-            'Local experiment stop logic is not implemented.')
+        subprocess.Popen([
+            '/bin/bash', '-c', 'docker rm --volumes --force '
+            '$(docker ps -f "name=r-' + str(experiment_name) + '-[0-9]+$" -q)'
+        ])
```

Comment:
why not subprocess.run()?

is /bin/bash -c necessary?

Reply:
> why not subprocess.run()?

I guess we prefer to the wrapper version in this code repo for a consistency, `common/new_process`, which adopts `subprocess` already. The current way is a solution to avoid some errors. But maybe we can add `FORCE_LOCAL` in the Makefile in default and use `new_process` back.

> is /bin/bash -c necessary?

Yes. Otherwise, it will look at the `$(docker ps ...)` as an argument, rather than executing it first and getting results.